### PR TITLE
Update Sleeveless Simon by lasermonkey12 showcase

### PR DESCRIPTION
**Pull request by nataliasayang**
Update with user ID and correcting design to Simone (not Simon).

Resolves https://github.com/freesewing/freesewing/issues/6668

### CODE REVIEW

File: markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 date: 20240106
 intro: "Maker lasermonkey12 has made lots of great Simon shirts, including this sleeveless one."
-designs: ["simon"]
-maker: Natalia
+designs: ["simone"]
+maker: lasermonkey12
+author: 22007
 ---
 
 Maker lasermonkey12 has made lots of great Simon shirts, including this sleeveless one. We love the fresh look, well-suited for hot weather! This was shared on [Discord](https://discord.freesewing.org/) and is reposted here with permission.
```
Comment (HaasJona) [inline markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md:11]: Should also say Simone here, then?

Comment (nataliasayang) [reply markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md:11]: Yes, please go ahead and make the change.

Comment (HaasJona) [reply markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md:11]: I don't think I have permissions to edit stuff in your pull request. :frowning_face:

Comment (nataliasayang) [reply markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md]: Ah, bad luck. I fixed it.

**Review by joostdecock — APPROVED** [review]

**Pull request merged**